doc/conventions.md

```diff
@@ -0,0 +1,46 @@
+# Code Style Conventions
+
+This document outlines the code style conventions used in the Clean project.
+
+## Standard Lean/Mathlib Conventions
+
+We follow the standard Lean and Mathlib conventions for most formatting:
+
+- **Type annotations**: Use spaces around `:` 
+  - ✅ `(F : Type)`
+  - ❌ `(F:Type)`
+
+- **Assignment operator**: Use spaces around `:=`
+  - ✅ `let x := 5`
+  - ❌ `let x:=5`
+
+- **General named parameters**: Use spaces around `:=`
+  - ✅ `(constant := constant)`
+  - ❌ `(constant:=constant)`
```

Comment:
this is inconsistent with the local convention below!

Reply:
fixed.